dist/bin/no-internal-report.js

```diff
@@ -1,64 +1,44 @@
 #! /usr/bin/env node
-
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 "use strict"
 const path = require('path');
-const fs = require('fs');
-
-function getNodeModulesDir(startPath) {
-  let currPath = startPath;
-  let baseName = path.basename(currPath);
-  while (baseName !== "node_modules") {
-    currPath = path.resolve(currPath, "..")
-    if (currPath === "/")
-      return undefined;
-    baseName = path.basename(currPath);
+const assert = require('assert');
+const eslintBinPath = path.join(require.resolve("eslint"), "../../bin/eslint.js");
+
+const args = { tags: ["alpha", "internal"], files: [] };
+for (let i = 2; i < process.argv.length; ++ i) {
+  if (process.argv[i] == '--tags') {
```

Comment:
Bit of a personal preference here I guess but perhaps more readable code if something like `const flag = process.argv[i]` here instead of accessing it every time it needs to be read

Reply:
changed